### DISCUSSION TITLE
No vyper plugin

### DISCUSSION THREAD

**Discussion by Philip-kay** [Cyfrin/moccasin-full-course-cu#3] [Q&A]
Hey, I'm new to smart contracts, but while following the video on vyper, I tried to install the plugin(vyper) but the search results isn't bringing out any vyper plugin like it did in the video. What should I do?

**Comment by PatrickAlphaC** [comment]
Hello! Can you send a screenshot of what you're seeing?

**Comment by Scofield-Idehen** [comment]
I believe you should check your network connection, confirm if you typed the right command, or finally, check that you are not on the same repo. 

Could you send a screenshot if none of these help?
